### ISSUE TITLE
Observations in Baengnyeong Island, South Korea tagged with country code NP

### ISSUE THREAD

**Issue by dnoesgaard** [inaturalist/inaturalist#2223]
While browsing records in North Korea, I noticed that records from Baengnyeong Island (in South Korea) show up with location "...Inchon, South Korea", but if you go to encompassing places, the country is listed as North Korea, e.g.

https://www.inaturalist.org/observations/21865011

This is also reflected in the DwC-A ingested by GBIF, while records are being flagged "Country coordinate mismatch ", e.g. https://www.gbif.org/occurrence/2235520450

**Comment by kueda** [comment]
Our Korean borders came from GADM, which is apparently incorrect. I fixed our borders and the observations and being reprocessed now. Hopefully this will be corrected in the next DwC-A we make for you folks.

**Comment by dnoesgaard** [comment]
Thanks Ken-ichi. I'll trigger a crawl and see how it looks.

**Comment by dnoesgaard** [comment]
Ok I guess the archive we just crawled was built before the borders were fixed. I'll have a look again next week...

**Comment by kueda** [comment]
Yeah, the DwC-A job kicks off on Thursday and... takes a while. Best to check on it next week. FWIW, the "Encompassing Places" for https://www.inaturalist.org/observations/21865011 lists South Korea now, so I think everything ok in our database.
